src/Microsoft.TemplateEngine.Edge/DefaultEnvironment.cs

```diff
@@ -18,11 +18,12 @@ public class DefaultEnvironment : IEnvironment
         private const int DefaultBufferWidth = 160;
         private readonly IReadOnlyDictionary<string, string> _environmentVariables;
 
-        public DefaultEnvironment() : this(FetchEnvironmentVariables())
+        public DefaultEnvironment() : this(tabCompletionMode: false, FetchEnvironmentVariables())
```

Comment:
please add it as the last parameter and optional to ensure backwards compatibility.